### PR TITLE
Show outdated message when new test is available, and disable review button immediately

### PR DESCRIPTION
**Pull request by tmeasday**
1. You can't accept changes when there is a next build
2. As soon as there's a new snapshot for the current story, we change the header to send you there.

## How to QA

1. Create a build with changes
2. Browse to the story with changes
3. Run second build:
  a. Review button should be disabled as soon as second build reaches the "snapshotting" state
  b. Header should prompt you to switch stories as soon as the story you are looking at is captured.

<details>
  <summary>📦 Published PR as canary version: <code>0.0.66--canary.87.fc95b12.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromaui/addon-visual-tests@0.0.66--canary.87.fc95b12.0
  # or 
  yarn add @chromaui/addon-visual-tests@0.0.66--canary.87.fc95b12.0
  ```
</details>

### CODE REVIEW

File: src/screens/VisualTests/BuildResults.tsx [hunk 1]
```diff
@@ -74,6 +74,14 @@ export const BuildResults = ({
...
     ),
   ];
+  const nextStoryTests = [
+    ...getFragment(
+      FragmentNextStoryTestFields,
+      "testsForStory" in nextBuild ? nextBuild.testsForStory.nodes : []
+    ),
+  ];
+  const isStoryOutdated =
+    !isReviewable && nextStoryTests.every(({ status }) => status !== TestStatus.InProgress);
```
Comment (ghengeveld) [inline src/screens/VisualTests/BuildResults.tsx:84]: Can't this be achieved by looking at `nextBuild.completedAt` or `nextBuild.result`?

File: src/screens/VisualTests/StoryInfo.tsx [hunk 2]
```diff
@@ -73,19 +76,17 @@ export const StoryInfo = ({
         </small>
       </Text>
     );
-  } else if (isOutdated) {
+  } else if (isStoryOutdated && switchToNextBuild) {
```
Comment (ghengeveld) [inline src/screens/VisualTests/StoryInfo.tsx:79]: Looks like we're changing the definition of 'outdated' from "there are code changes, you need a new build" to "there's a new build, you need to apply it". I'd rather use different terminology, e.g. "stale".

**Review by ghengeveld — APPROVED** [review]

File: src/screens/VisualTests/BuildResults.tsx (hunk 1 above)
Comment (tmeasday) [reply src/screens/VisualTests/BuildResults.tsx:84]: We aren't waiting for the whole build to finish, just all the tests for the current story.

File: src/screens/VisualTests/StoryInfo.tsx (hunk 2 above)
Comment (tmeasday) [reply src/screens/VisualTests/StoryInfo.tsx:79]: Superceded?

**Comment by tmeasday** [comment]
@MichaelArestad I accidentally merged this but I was still hoping to resolve this final question: https://www.chromatic.com/review?appId=6480e1b0042842f149cfd74c&number=87&type=linked

**Pull request merged**